### PR TITLE
Clean up logging and make new output

### PR DESCRIPTION
**Pull request by RobertManningSmith**
Created a neater formatting of logging where one function corresponds to one output file. Also created two additional output files, percent of cases per admin zone that are fatal. Percent of people in admin zone who died of covid.

### CODE REVIEW

File: Protecs/src/main/java/uk/ac/ucl/protecs/sim/Logging.java
Comment (swise5) [inline Protecs/src/main/java/uk/ac/ucl/protecs/sim/Logging.java:1]: I'm still a little concerned by this hard-coding (I'd much prefer an external file where we could specify age categories, birthrates, names for these groups, etc) but this is a huge improvement over before - and the commenting is great, thanks!

**Review by swise5 — APPROVED** [review]
This all looks to me as if it's moving in a good direction! The greater abstraction is really nice. 

I'd be curious to hear how you're finding running this at scale in terms of the relative processing power needed with all this logging. If it's becoming an issue, it may ultimately be worth exploring exporting the data relative to the Infection objects and then post-processing them. For example, we might export a simple case as follows:

dummyoutput.txt
> host source start sympt died cured
> p1     null      0:d1      4:d1         7:d1          -1:null
> p2     p1        3:d1      7:d1        -1:null          11:d1
> p3     p1        3:d1     6:d2         9:d2          -1
> p4     p3       5:d2     -1:null        -1:null          7:d2

Showing that P1 was patient 0, was symptomatic by day 4 in District 1, and died on day 7 in District 1; P4 contracted the infection from contact with P3 on day 5 in District 2; etc. So then for cases in each of the regions, we could reconstruct/combine as desired.

The nice thing about this is it would require minimum sorting/streaming/processing etc, and after the fact we could slice and dice as desired. But if it's working fine, I don't think that's a priority!

**Comment by swise5** [comment]
Looks good to me!

**Comment by swise5** [comment]
Hey @RobertManningSmith! I'm going to hold off on deleting the branch because of the unmerged commit until you let me know how you'd like me to proceed!

**Comment by RobertManningSmith** [comment]
Reopening as this didn't get merged into main! I'm not sure what this unmerged commits thing is, there isn't anything on my end that hasn't been pushed through.

**Pull request merged**